src/crypto/store/indexeddb-crypto-store.js

```diff
@@ -110,15 +110,17 @@ export default class IndexedDBCryptoStore {
             const req = this._indexedDB.deleteDatabase(this._dbName);
 
             req.onblocked = () => {
-                reject(new Error(
-                    "unable to delete indexeddb because it is open elsewhere",
-                ));
+                console.log(
+                    `can't yet delete IndexedDBCryptoStore because it is open elsewhere`,
+                );
             };
 
             req.onerror = (ev) => {
-                reject(new Error(
-                    "unable to delete indexeddb: " + ev.target.error,
-                ));
+                // in firefox, with indexedDB disabled, this fails with a
+                // DOMError. We treat this as non-fatal, so that people can
+                // still use the app.
+                console.warn(`unable to delete IndexedDBCryptoStore: ${ev.target.error}`);
```

Comment:
possibly, but how would we implement that, given this will happen on a different MatrixClient to subsequent access?